internal/storage/sql/common/flag.go

```diff
@@ -34,16 +34,21 @@ func emptyAsNil(str string) *string {
 }
 
 // GetFlag gets a flag with variants by key
-func (s *Store) GetFlag(ctx context.Context, key string) (*flipt.Flag, error) {
+func (s *Store) GetFlag(ctx context.Context, namespaceKey, key string) (*flipt.Flag, error) {
+	if namespaceKey == "" {
+		namespaceKey = storage.DefaultNamespace
+	}
+
 	var (
 		createdAt fliptsql.Timestamp
 		updatedAt fliptsql.Timestamp
 
 		flag = &flipt.Flag{}
 
-		err = s.builder.Select("\"key\", name, description, enabled, created_at, updated_at").
+		err = s.builder.Select("namespace_key, \"key\", name, description, enabled, created_at, updated_at").
 			From("flags").
-			Where(sq.Eq{"\"key\"": key}).QueryRowContext(ctx).Scan(
+			Where(sq.And{sq.Eq{"namespace_key": namespaceKey}, sq.Eq{"\"key\"": key}}).QueryRowContext(ctx).Scan(
```

Comment:
Take it or leave it: I think you can do this in a few places:
```suggestion
			Where(sq.Eq{"namespace_key": namespaceKey, \"key\"": key}).QueryRowContext(ctx).Scan(
```